samples/python/visualization_openGL.py

```diff
@@ -15,7 +15,7 @@ def increaseTemp():
         global temperature
         temperature += 0.1
         system.thermostat.set_langevin(kT=temperature, gamma=1.0)
-        print system.temperature 
+        print temperature 
```

Comment:
`system.temperature` contains the current temperature, while `temperature` is the target temperature. I think it makes more sense to print the current one.